src/com/booksaw/betterTeams/commands/HelpCommand.java

```diff
@@ -67,7 +64,9 @@ public void fullyCustom(CommandSender sender, String label) {
 
 		if (!f.exists()) {
 			try {
-				f.createNewFile();
```

Comment:
Check the javadocs: https://docs.oracle.com/javase/7/docs/api/java/io/File.html#createNewFile()
False is returned if the file already exists which will never happen. 
An error is thrown if the file could not be created (caught in the lower catch)